roles/host-provision/tasks/config-tasks.yml

```diff
@@ -17,10 +17,46 @@
   ping:
   tags: host-provision
 
-# Placeholder for the core config tasks to be executed by the host-provision.sh utility
-# - name: config-tasks | Create LVM layer
-# ...
 
+# Not including ntp set up, as we already have code path built into `base-provision` role
+# that processes the commmand line switch: `--ntp-pref`
 
-# - name: config-tasks | For RHEL, subscription manager
-# ...
+
+# LVM creation
+# The net result will be that `/dev/mapper/db-sw` can be input into `install-oracle.sh` for local u01 storage
+- name: config-tasks | Create LVM layer
+  # vgcreate <VG> <block device WWID>
+  shell: vgcreate db {{ u01_lun }}
+  when:
+    - u01_lun | length > 0    
+  tags: host-provision
+
+- name: Create logical volume
+  # lvcreate -l 100%FREE -n <LV> <VG>
+  shell: lvcreate -l 100%FREE -n sw db
+  when:
+    - u01_lun | length > 0    
+  tags: host-provision
+
+
+# RHEL specific: subscription manager
+- name: Remove registrations
+  become: true
+  redhat_subscription:
+    state: absent
+  when:
+    - ansible_distribution == 'RedHat'
+  tags: host-provision
+
+- name: Register host
+  become: true
+  redhat_subscription:
+    state: present
+    username: "{{ rhel_username }}"
+    password: "{{ rhel_password }}"
+    auto_attach: true
+  # we ignore when: "This system is already registered" is in the output
```

Comment:
Can we please only ignore "This system is already registered" errors?  I'd imagine that many other error cases would warrant issues.